cluster/gce/upgrade.sh

```diff
@@ -439,6 +439,17 @@ function do-node-upgrade() {
   echo "== Finished upgrading nodes to ${KUBE_VERSION}. ==" >&2
 }
 
+function update-coredns-config() {
+  # Deletes the existing CoreDNS ConfigMap so that the latest can be applied.
+  echo "== Deleting existing CoreDNS ConfigMap. ==" >&2
+  "${KUBE_ROOT}/cluster/kubectl.sh" delete configmap coredns --namespace=kube-system 2>/dev/null
+
+  echo "== Waiting for CoreDNS to get the latest ConfigMap. ==" >&2
```

Comment:
I think this should say "Waiting for Addon Manager to restore CoreDNS ConfigMap"